ably/realtime_conn_spec_integration_test.go

```diff
@@ -2707,9 +2624,18 @@ func TestRealtimeConn_RTN19b(t *testing.T) {
 			Action:  ably.ActionDetach,
 		},
 	}
-	for _, expect := range msgs {
+	var gots []*ably.ProtocolMessage
+	for range msgs {
 		var got *ably.ProtocolMessage
 		ablytest.Instantly.Recv(t, &got, out, t.Fatalf)
+		gots = append(gots, got)
+	}
+
+	sort.Slice(msgs, func(i, j int) bool { return  msgs[i].Action < msgs[j].Action})
```

Comment:
```suggestion

```
I'm not a fan of changing expected values as its usually a code smell in unit tests. In this case however I don't think this line is needed at all because `msgs` is hardcoded in a way that is already sorted by action order. Suggest deleting this line.